OpenTrons/opentrons-api
# fix(app-shell, app-shell-odd): Prevent excessive notify logs

Closes [RQA-2431](https://opentrons.atlassian.net/browse/RQA-2431)



# Overview

Because subscription logic is handled by components currently, logs are excessive in some spots, especially on the OT-2. Let's comment out the logs until we refactor subscription logic to be handled directly by the robot-server (next release). Note that there aren't any excessive network requests - it's just logs.


# Test Plan
- Start a dev build and look at the console. 
- Navigate to an OT-2 and start a pipette flow, run a protocol, etc. 
- Verify that there are no [notify] logs from the app-shell.  


# Changelog
- Removed spammy notification logs. 




# Risk assessment
low



[RQA-2431]: https://opentrons.atlassian.net/browse/RQA-2431?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
